pkg/kine/drivers/sqlite/schema.go

```diff
@@ -13,10 +14,47 @@ import (
 // already
 
 const (
-	databaseSchemaMajorVersion = 1
-	databaseSchemaMinorVersion = 0
+	databaseSchemaMajorVersion int16 = 0
+	databaseSchemaMinorVersion int16 = 1
+	mask                       int32 = 0xFFFF // Mask for 2 bytes (16 bits)
```

Comment:
This package-wide variable should be removed or renamed with a more specific name.

Reply:
it's removed :)